proto.go

```diff
@@ -27,15 +27,7 @@
 
 package clouditor
 
-// Run owl2proto to generate the ontology proto file. The tools needs as arguments the following:
-// - root-resource-name from the owl file
-// - owl file in the owx format
-// - header-file
-// - output-path for the proto file (optional, default is "api/ontology.proto")
-//go:generate owl2proto generate-proto --root-resource-name=https://ontology.emerald-he.eu/classes/Resource internal/ontology/ontology.owx --header-file=internal/ontology/clouditor_header.proto --output-path=api/ontology/ontology.proto --full-semantic-mode=false --deterministic-field-numbers=true
 //go:generate buf format -w
```

Comment:
We do not need that anymore. That was used for the generated ontology.proto file.